examples/ota-provider-app/linux/README.md

```diff
@@ -10,13 +10,81 @@ Suggest doing the following:
 
 ## Usage
 
-`./ota-provider-app [-f/--filepath \<file\>]`
+`./ota-provider-app [-f/--filepath \<file\>] [-o/--otaImageList \<file\>]`
 
 If `--filepath` is supplied, `ota-provider-app` will automatically serve that
 file to the OTA Requestor (SoftwareVersion will be Requester version + 1).
 
-If no `--filepath` is supplied, `ota-provider-app` will respond to `QueryImage`
-with `NotAvailable` status.
+If `--otaImageList` is supplied, `ota-provider-app` will parse the JSON file and
+extract all required data. Then the most recent, valid software version will be
+selected and the corresponding ota-file will be sent to the OTA Requestor.
+
+
+Here's an example of the otaImageList file contents:
+
+{
+    "foo": 1, // ignored by parser
```

Comment:
Is this intentional?

Reply:
Yes, I wanted to make it clear that any other JSON keys will be ignored by the parser.